### PR TITLE
Changed Kusto schema 2019-05-15

### PR DESCRIPTION
**Pull request by oflipman**

### CODE REVIEW

**Comment by oflipman** [comment]
Hi.
We have added a few features in our swagger file, and we want to align them with the schema

**Comment by anthony-c-martin** [comment]
@oflipman is your team interested in [onboarding to the autogeneration pipeline](https://github.com/Azure/azure-resource-manager-schemas/blob/master/generator/README.md#onboarding-teams-to-autogeneration-pipeline)? This will result in schemas being generated from your swagger specs automatically, removing the need for manual changes like this.

@ifeoluwaokunoren FYI

**Comment by oflipman** [comment]
Hi @anthony-c-martin, thanks for the information.
We will probably do that next time.
For this PR, I prefer to use the one that we built

**Comment by anthony-c-martin** [comment]
> Hi @anthony-c-martin, thanks for the information.
> We will probably do that next time.
> For this PR, I prefer to use the one that we built

Sure, will take a look.

File: schemas/2019-05-15/Microsoft.Kusto.json [hunk 1]
```diff
@@ -179,303 +182,381 @@
...
+        },
+        "tableName": {
+          "type": "string",
+          "description": "The table where the data should be ingested. Optionally the table information can be added to each message."
+        },
+        "mappingRuleName": {
+          "type": "string",
+          "description": "The mapping rule to be used to ingest the data. Optionally the mapping information can be added to each message."
+        },
+        "dataFormat": {
```
Comment (anthony-c-martin) [inline schemas/2019-05-15/Microsoft.Kusto.json]: You need to include "oneOf" for every single property in this file to reference https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#/definitions/expression. This allows customers to use template expressions when working with this resource type.

**Comment by oflipman** [comment]
Hi @anthony-c-martin is the fix ok?

File: schemas/2019-05-15/Microsoft.Kusto.json [hunk 2]
```diff
@@ -1,481 +1,593 @@
...
-            },
-            "dataManagementPublicIpId": {
-              "description": "Data management's service public IP address resource id.",
-              "type": "string"
-            }
-          },
-          "description": "A class that contains virtual network definition."
-        }
-    }
+	"id": "https://schema.management.azure.com/schemas/2019-05-15/Microsoft.Kusto.json#",
```
Comment (anthony-c-martin) [inline schemas/2019-05-15/Microsoft.Kusto.json]: Please can you change tabs back to 2 space indents?

**Pull request merged**